contracts/callback/CallbackUtils.sol

```diff
@@ -160,14 +171,16 @@ library CallbackUtils {
         bytes32 key,
         Withdrawal.Props memory withdrawal,
         EventUtils.EventLogData memory eventData
-    ) internal {
+    ) external {
         if (!isValidCallbackContract(withdrawal.callbackContract())) { return; }
 
         validateGasLeftForCallback(withdrawal.callbackGasLimit());
 
+        EventUtils.EventLogData memory withdrawalData = WithdrawalEventUtils.createEventData(withdrawal, Withdrawal.WithdrawalType.Normal);
```

Comment:
i think passing Withdrawal.WithdrawalType.Normal is not accurate here since this could be a callback for shift or glv?